doc-content/enterprise-only/bpmn/bpmn-business-rule-task-ref.adoc

```diff
@@ -1,5 +1,4 @@
 [id='bpmn-business-rule-task-ref_{context}']
-
 = Business rule task 
```

Comment:
Not in this PR, but Figure 4.1 Task needs a header. Suggest "Task type" and "Notation", or something. That's similar to how it is in the DMN spec.

Also, search the doc for "In additions,", which I think is a typo for "In addition,".